.github/workflows/lint.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             go.sum
       - uses: golangci/golangci-lint-action@v3.7.0
         with:
-          version: v1.52.2
+          version: v1.54.2
```

Comment:
Confirming this is the latest release: https://github.com/golangci/golangci-lint/releases/tag/v1.54.2

https://github.com/golangci/golangci-lint/releases/tag/v1.54.0 introduced Go 1.21 support.